Tests/OptimizelyTests-Common/DecisionServiceTests_Features.swift

```diff
@@ -253,10 +301,58 @@ extension DecisionServiceTests_Features {
         XCTAssertNil(variation)
     }
     
-    func testGetVariationForFeatureRolloutMultiple() {
-        // add tests for last rollout handling
+    func testGetVariationForFeatureRolloutFallbackRule() {
+        self.config.project.rollouts = [try! OTUtils.model(from: sampleRolloutData)]
+        self.config.project.rollouts[0].experiments[0].trafficAllocation[0].endOfRange = 0
+        featureFlag.rolloutId = kRolloutId
+        self.config.project.featureFlags = [featureFlag]
+        
+        let variation = self.decisionService.getVariationForFeatureRollout(config: config,
+                                                                           featureFlag: featureFlag,
+                                                                           userId: kUserId,
+                                                                           attributes: kAttributesAgeMatch)
```

Comment:
With this setting, we cannot tell if it skips the rest of them or goes thru all rules. What about this setting:

Rule1: age < 30 (traffic allocation = 0)
Rule2: age < 40 (traffic allocation = 10000)
Rule3: everybody

When attribute age = 20, we should expect variation for Rule 3 if it skips. Otherwise, variation for Rule 2 will hit.